sys/psa_crypto/Makefile.dep

```diff
@@ -0,0 +1,236 @@
+ifneq (,$(filter psa_crypto,$(USEMODULE)))
+  USEMODULE += random
+endif
+
+# Asymmetric
+ifneq (,$(filter psa_asymmetric,$(USEMODULE)))
+  USEMODULE += psa_key_slot_mgmt
+endif
+
+## ECC_P192R1 backend
+ifneq (,$(filter psa_asymmetric_ecc_p192r1,$(USEMODULE)))
+  ifeq (,$(filter psa_asymmetric_ecc_p192r1_custom_backend,$(USEMODULE)))
+    DEFAULT_MODULE += psa_asymmetric_ecc_p192r1_backend_microecc
+    ifneq (,$(filter periph_ecc_p192r1,$(FEATURES_USED)))
+      USEMODULE += psa_asymmetric_ecc_p192r1_backend_periph
+      DISABLE_MODULE += psa_asymmetric_ecc_p192r1_backend_microecc
+    else
+      FEATURES_OPTIONAL += periph_ecc_p192r1
+    endif
+  endif
+endif
+
+ifneq (,$(filter psa_asymmetric_ecc_p192r1_backend_microecc, $(USEMODULE)))
+  USEPKG += micro-ecc
+  USEMODULE += psa_uecc
+  USEMODULE += psa_uecc_p192
+endif
```

Comment:
Since the `psa_asymmetric_ecc_p192r1_backend_microecc` is only selected in `DEFAULT_MODULE`, which only gets evaluated on the last makefile dep run, it never actually appears in `USEMODULE` thus never selecting these modules and packages...

One can get around the issue with explicitly adding all deps in as `DEFAULT_MODULES` such as:

```diff
--- a/sys/psa_crypto/Makefile.dep
+++ b/sys/psa_crypto/Makefile.dep
@@ -11,9 +11,14 @@ endif
 ifneq (,$(filter psa_asymmetric_ecc_p192r1,$(USEMODULE)))
   ifeq (,$(filter psa_asymmetric_ecc_p192r1_custom_backend,$(USEMODULE)))
     DEFAULT_MODULE += psa_asymmetric_ecc_p192r1_backend_microecc
+    USEPKG += micro-ecc
+    DEFAULT_MODULE += psa_uecc
+    DEFAULT_MODULE += psa_uecc_p192
     ifneq (,$(filter periph_ecc_p192r1,$(FEATURES_USED)))
       USEMODULE += psa_asymmetric_ecc_p192r1_backend_periph
       DISABLE_MODULE += psa_asymmetric_ecc_p192r1_backend_microecc
+      DISABLE_MODULE += psa_uecc
+      DISABLE_MODULE += psa_uecc_p192
     else
       FEATURES_OPTIONAL += periph_ecc_p192r1
     endif
```

The problem now is that there is a `USEPKG` as a `DEFAULT_MODULE` dep.  Currently there is no `DEFAULT_PKG` and `DISABLE_PKG`... If you implement it though you further issues of `Error - using unknown modules` as the pkg has its own dependencies on modules that may not get found or something...